manage.py

```diff
@@ -140,6 +140,56 @@ def _add_live_test_subparser(subparsers):
     )
 
 
+def _add_validate_schema_subparser(subparsers):
+    """Add the validate-schemas subparser: manage.py validate-schemas [options]"""
+    schema_validation_usage = 'manage.py validate-schemas [options]'
+    schema_validation_description = ("""
+StreamAlertCLI v{}
+Run end-to-end tests that will attempt to send alerts
+
+Available Options:
+
+    --test-files         Name(s) of test files to validate, separated by spaces (not full path)
```

Comment:
Yes I agree with you here, but argparse doesn't :) List supplied as args to argparse are expected to be space delimited. We could change this to expect a string that is a comma separated and handle all of the logic in code, but then we lose the validation/err handling baked into argparse. At one point (with the --rules flag) I was doing comma separated but we decided to do spaces since it was safer with less overhead